presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/TestElasticsearch7IntegrationSmokeTest.java

```diff
@@ -20,7 +20,7 @@
 {
     public TestElasticsearch7IntegrationSmokeTest()
     {
-        super("elasticsearch:7.0.0");
+        super("docker.elastic.co/elasticsearch/elasticsearch:7.0.0");
```

Comment:
why it changed?

Reply:
Indeed. If I recall correctly, the images from docker.elastic.co are the commercial images, so the choice to use the official docker images from dockerhub was intentional.